### PR TITLE
[FEAT] Added Notification and error handling

### PR DESCRIPTION
**Pull request by OmKakatkar**
* Adds Error Handling to API Calls
* Notifies user about add and remove of item and errors
* Adds 404 Pages

### CODE REVIEW

**Review by Hnh013 — APPROVED** [review]
code is well written and clean , good to merge

File: src/util/product-request.js [hunk 1]
```diff
@@ -1,87 +1,158 @@
...
+	} catch (err) {
+		notify(error, 'Unable to remove data');
+		console.error(err);
+	}
+};
+
+const changeCartQuantity = async (authToken, product, type) => {
+	try {
+		const data = await axios.post(
+			`${API_CART}/${product._id}`,
```
Comment (rahulyadav139) [inline src/util/product-request.js:135]: Great you have saved the API path in a variable. What you can do you can save it as environmental variables.

**Review by rahulyadav139 — APPROVED** [review]

File: src/util/product-request.js (hunk 1 above)
Comment (OmKakatkar) [reply src/util/product-request.js:135]: Thanks for the suggestion. Will keep that as a future optimization

**Review by lomesshh — APPROVED** [review]
clean and modular code
404 page working as expected

looks good to merge

**Pull request merged**